src/iio_wrapper.cpp

```diff
@@ -89,6 +89,9 @@ int32_t IIOWrapper::m_count = 0;
 
 std::mutex IIOWrapper::m_mutex;
 
+std::string IIOWrapper::s_device_name = "";
+std::string IIOWrapper::s_device_trigger_name = "";
```

Comment:
A small observation: no need to initialize std::string with "" since by default it will be constructed that way.